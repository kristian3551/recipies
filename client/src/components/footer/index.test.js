import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

describe('<Footer/>', () => {
    it('should render properly', () => {
        const component = renderer.create(<Footer/>);
        const node = component.toJSON();
        expect(node).toMatchSnapshot();
    });
})