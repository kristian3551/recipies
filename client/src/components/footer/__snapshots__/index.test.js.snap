// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> should render properly 1`] = `
<footer
  className="mastfoot mt-5 text-center text-light"
>
  <div
    className="inner"
  >
    <p>
      Made with â™¥ by 
      <a
        className="footlink"
        href="/"
      >
        Recipes
      </a>
      .
    </p>
  </div>
</footer>
`;
